docs/setup_guides/hpc.rst

```diff
@@ -34,12 +34,19 @@ After you have logged into your HPC system, download and install Miniconda:
 This contains a self-contained Python environment that we can manipulate
 safely without requiring the involvement of IT. It also allows you to
 create isolated software environments so that we can experiment in the
-future safely. Before creating your environment, we recommend you update
-your conda package manager with
+future safely. 
+
+Before creating your environment, update your conda package manager with packages 
+from the conda-forge channel instead of the default channel and install Mamba, 
+which works like conda but is written in C++ and therefore creates environments faster. 
 
 ::
     
-    conda update conda
+    conda config --add channels conda-forge --force
+    conda config --remove channels defaults --force 
```

Comment:
I'm not sure why you need to use only conda-forge? This is to ensure the best compatibility between all the dependencies. This config might prevent using other channels like pytorch, don't this? 
But at this point if a user needs other channels he knows what he's doing, so maybe that's OK.

Reply:
Any kind of channel mixing is problematic. We always recommend to use only 1 and, if it is extremely necessary, add the extra channel in the env file configuration. However, the ideal of keeping defaults and conda-forge is usually problematic, specially without strict channel policy. That is b/c those two channels overlap a lot. It is not like getting pytorch from another channel, it is like mixing from two different sources for all packages.

The other advantage, besides avoiding compatibility issues, is to speed up the solver time by reducing the amount of packages it has to work with.

Reply:
Thanks for the clarification Filipe!  This has been a moving target for many of us, so good to know that one channel is the way to go now!